docs/howto/manage-web-app-charms/use-a-database.rst

```diff
@@ -0,0 +1,335 @@
+.. _use-db-with-12-factor-charms:
+
```

Comment:
@alithethird We've begun adding [meta descriptions](https://canonical-starflow.readthedocs-hosted.com/how-to/add-a-page-meta-description/) to new pages. Would you feel up to writing one?

Reply:
You're very correct. Sorry, I forgot that page is private. I'll suggest one. :)